src/main/java/com/movie_collection/be/Movie.java

```diff
@@ -3,7 +3,8 @@
 import javafx.beans.property.StringProperty;
 
 import java.sql.Date;
+import java.util.List;
 
-public record Movie(int id, StringProperty name, double rating, StringProperty path, Date lastview) {
+public record Movie(int id, StringProperty name, double rating, StringProperty path, List<Category> categories, Date lastview) {
```

Comment:
can we please maybe discuss if path is the correct naming here since it is defined absolute path 
its more like good practise I recon

Reply:
this is only up to us so feel free to suggest what is better from your view. 
Personally, I don't think I would differ between the two only based on variable name.

Reply:
we could call it absolute_path / absolutePath ? 
we are becoming very sensitive to naming haha

Reply:
https://github.com/TomassSimko/Private-Movie-Collection/pull/9/commits/14259bdd4e4462866de72177f6a3d1699b5f6032